### PR TITLE
Add more newcommer friendly guides to README

### PR DESCRIPTION
**Pull request by chungngoops**

### CODE REVIEW

**Comment by lauracowen** [comment]
Many years ago, we had a “back to basics: debugging” article on WASdev website. That was about WDT tools but some of it might(or might not) be useful. Here it is for due diligence:
[index.txt](https://github.com/OpenLiberty/start.openliberty.io/files/13299423/index.txt)

Not an opinion on what to do about the starter readme, just adding info for now.

**Comment by yeekangc** [comment]
General comments not specifically for this PR but more in general for the README ...

- Useful to have an introduction about the project and what the repository is for (besides what is under About that GH shows)?

- Helpful to note these as guidance for developers looking to contribute to the project?

- See how we can link to or reconcile with the content in CONTRIBUTING.md, which is typically where we describe how one can contribute?

@natalie-bernhard @steven1046, please consider.  ^^^

Again, @chungngoops, thank you for the PR.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
 # Open Liberty Starter
 
-## How to deliver code and test
+## How to run locally in dev mode
+1) Clone the repository
+2) Run `cd start.openliberty.io`
+3) Run `mvn liberty:dev` 
+4) Then you can start testing all endpoints at `https://localhost:9443/api/`   
+
+## How to debug using IntelliJ
```
Comment (yeekangc) [inline README.md:9]: We can reference the use of Liberty Tools and what it provides for debugging?  Cc @TrevCraw @aparnamichael

Comment (chungngoops) [reply README.md:9]: not sure if it's only me, the Liberty Toools doesn't work for me, I get this message:
`No Liberty Maven or Liberty Gradle projects detected in this workspace.`

**Comment by chungngoops** [comment]
> Many years ago, we had a “back to basics: debugging” article on WASdev website. That was about WDT tools but some of it might(or might not) be useful. Here it is for due diligence: [index.txt](https://github.com/OpenLiberty/start.openliberty.io/files/13299423/index.txt)
> 
> Not an opinion on what to do about the starter readme (sounds like the info would be useful but I'm not sure yet how/where best to present it), just adding info for now.

Could you please provide the link to that article ? maybe we can put the link in the guide.

**Comment by lauracowen** [comment]
> > Many years ago, we had a “back to basics: debugging” article on WASdev website. That was about WDT tools but some of it might(or might not) be useful. Here it is for due diligence: [index.txt](https://github.com/OpenLiberty/start.openliberty.io/files/13299423/index.txt)
> > Not an opinion on what to do about the starter readme (sounds like the info would be useful but I'm not sure yet how/where best to present it), just adding info for now.
> 
> Could you please provide the link to that article ? maybe we can put the link in the guide.

See [link](https://github.com/OpenLiberty/start.openliberty.io/files/13299423/index.txt) in my comment above. It's not online any more.

**Comment by chungngoops** [comment]
I added a new commit which move the setup guide to CONTRIBUTING.md and provide the link how to run/debug using the Liberty Tool. I also re-write the README.md, if anyone have a better introduction to put in README please post it here so I can put it in that file.
@lauracowen : the debuging guide using WDT tools misses many original images, I'm not sure we should provide the plain text version?

**Review by steven1046 — APPROVED** [review]

**Comment by yeekangc** [comment]
Thank you, @chungngoops!

**Pull request merged**